### ISSUE TITLE
GUI - show numeric locale in About dialogue

### ISSUE THREAD

**Issue by GoogleCodeExporter** [shawnlaffan/biodiverse#274]
```
The numeric locale is a source of grief for some users, as valid numbers in one 
locale will be invalid in others.  

The system prints the locale to the log window in startup, but this disappears 
off screen pretty quickly.

Adding it to the About dialogue will help with diagnosing problems.  
```

Original issue reported on code.google.com by `shawnlaffan` on 21 Nov 2011 at 3:40

**Comment by shawnlaffan** [comment]
This has possibly been obviated by 9fc69fea969258e3679503d090e5cc3a43f40f54

**Comment by shawnlaffan** [comment]
Closing under the assumption it is fixed by 9fc69fea969258e3679503d090e5cc3a43f40f54
